### PR TITLE
Add additional doc for using AMS with POROs

### PR DESCRIPTION
**Pull request by tommi-lew**
#### Purpose

For additional clarity.
#### Related GitHub issues

https://github.com/rails-api/active_model_serializers/issues/1854

### CODE REVIEW

File: docs/howto/serialize_poro.md [hunk 1]
```diff
@@ -29,4 +29,10 @@ class MyModel < ActiveModelSerializers::Model
...
 ```
 
-The default serializer would be `MyModelSerializer`.
\ No newline at end of file
+The default serializer would be `MyModelSerializer`:
+```ruby
+# my_model_serializer.rb
+class MyModelSerializer < ActiveModel::Serializer
+  attributes :id, :name, :level
+end
```
Comment (bf4) [inline docs/howto/serialize_poro.md]: Would you mind commenting here on how this change improves the documentation? There's definitely room for improvement, in particular, I don't think the word `default` is helpful.  It's more like the automatic lookup would resolve to a serializer with that name.  Besides that, how do you see adding an example serializer helpful?  I'd rather have a link to the serializers docs for more info.

Comment (DrSayre) [inline docs/howto/serialize_poro.md]: @tommi-lew @bf4 I'm fine with any changes to this. I put that sentence there to point out the serializer is the same name as one created with ActiveRecord.

Comment (bf4) [inline docs/howto/serialize_poro.md]: Would you have thought otherwise?

**Comment by NullVoxPopuli** [comment]
@DrSayre do you think you can finish this up?

**Comment by richmolj** [comment]
I'm kinda down on this and the current documentation. We're encouraging people to use `ActiveModelSerializers::Model`, which most serializers do not need. Most serializers only require: 

```ruby
alias :read_attribute_for_serialization :send
```

And if we made an internal change they wouldn't even need that.

The exceptions are `ErrorSerializer` and caching. I suggest the documentation change be this - otherwise people (like me) are needlessly adding stuff to their POROs that is not needed.

**Comment by bf4** [comment]
Closing as stale

**Pull request closed without merging**